packages/workbox-cli/src/lib/logger.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+**/
+
+const chalk = require('chalk');
+
+module.exports = {
+  debug: (...args) => console.log(chalk.gray(...args)),
+  log: (...args) => console.log(...args),
+  warn: (...args) => console.warn(chalk.red(...args)),
```

Comment:
Yellow is a more comment color choice for warning and would match what we've done in the browser.